ci-operator/config/syndesisio/syndesis-qe/syndesisio-syndesis-qe-1.15.x__fuse-online-ocp4.13-lp-interop.yaml

```diff
@@ -0,0 +1,67 @@
+base_images:
+  cli:
+    name: "4.13"
+    namespace: ocp
+    tag: cli
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.19
+images:
+- context_dir: docker
+  dockerfile_path: Dockerfile
+  to: fuse-online-test-runner
+- dockerfile_literal: |-
```

Comment:
What are these two docker_file literals being used for?

It looks like you are creating two images
1. syndesis-src
2. syndesis-build

but you don't use them anywhere